app/application.js

```diff
@@ -1,18 +1,53 @@
 import Mn from 'backbone.marionette';
 import Backbone from 'backbone';
-import tracksView from 'views/tracks';
+import Tracks from './collections/tracks';
+import Playlists from './collections/playlists';
+import AppLayout from './views/app_layout';
 
+require('./styles/app.styl');
 
-class Application extends Mn.Application {
+const Application = Mn.Application.extend({
 
-    onStart () {
-        console.log('start');
+    initialize: function () {
+        this.allTracks = new Tracks([], { type: 'all' });
+        this.upNext = new Tracks();
+        this.upNext.on('change:currentTrack', function() {
+            const upNext = app.upNext;
```

Comment:
You often use `const`. Theorically, `const` are only here for _constants_ (like `const PI=3.14`) not for vars. Prefer the use of `let` in your blocks instead.